docs/unified-test-documentation/test-matrix.md

```diff
@@ -0,0 +1,148 @@
+# Test matrix
+
+## About
+
+<!--
+Matryca testowa służy do określenia które z zetsawów i przypadków testych
+opisanych w dokumentacji testowej dedykowane są dla danej platformy
+-->
+
+## Test matrix - ASUS KGPE-D16
+
+### Module: dasharo-compatibility
+
```

Comment:
Lets use `<center></center>` to center the table

Reply:
OK